<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fWorkers</class>
 <widget class="QWidget" name="fWorkers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequrity</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Report:</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>331</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Params:</string>
   </property>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Auditories:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Workers:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_auditories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_workers">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_auditorylog">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Auditory Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_workerslog">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Workers Log</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
